packages/create-svelte/scripts/build-templates.js

```diff
@@ -0,0 +1,154 @@
+import fs from 'fs';
+import path from 'path';
+import parser from 'gitignore-parser';
+import prettier from 'prettier';
+import { transform } from 'sucrase';
```

Comment:
What's the reason to use `sucrase` here? If only TS is transpiled, we don't need it and can use `ts.transpileModule` instead.

Reply:
Sucrase preserves the original formatting better. If you use TS directly, this...

```svelte
<script lang="ts">
	let count = 0;

	const increment = () => {
		count += 1;
	};
</script>
```

...gets turned into this:

```svelte
<script>
	let count = 0;
	const increment = () => {
		count += 1;
	};
</script>
```

It's a little thing but it matters

Reply:
Didn't know that, makes sense. Maybe we should add a code comment then for future devs who think the same as me did first.

Reply:
good point, have done that